src/components/CurrencyRate.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import React, { PureComponent } from "react";
+import { View, StyleSheet } from "react-native";
+import Icon from "react-native-vector-icons/dist/Feather";
+import { BigNumber } from "bignumber.js";
+import type { Currency } from "@ledgerhq/live-common/lib/types/currencies";
+import colors from "../colors";
+import LText from "./LText";
+import CounterValue from "./CounterValue";
+
+type Props = {
+  currency: Currency & { color: string },
+  fontStyle?: *,
+  iconSize?: number,
+};
+
+class CurrencyRate extends PureComponent<Props> {
+  static defaultProps = {
+    iconSize: 10,
+  };
+
+  render() {
+    const { currency, fontStyle, iconSize } = this.props;
+    const one = new BigNumber(10 ** currency.units[0].magnitude);
+    return (
+      <View style={styles.wrapper}>
+        <Icon name="activity" color={currency.color} size={iconSize} />
+        <LText tertiary style={[styles.text, fontStyle]}>
+          {`${currency.ticker} 1 = `}
```

Comment:
uses a `<CurrencyUnitValue>` because the formatting changes depending on the languages.

Reply:
oh i see